Dockerfile.rhel7

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
If this is this identical to the Dockerfile won't a symbolic link suffice?

Reply:
They are not identical, the base images are different.